lib/pinpoint_supported_countries.rb

```diff
@@ -10,15 +10,6 @@ class PinpointSupportedCountries
   PINPOINT_VOICE_URL = 'https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-voice-countries.html'.freeze
 
   # The list of countries where we have our sender ID registered
```

Comment:
we should prob remove this comment too? since the thing it's attached to is gone
```suggestion
```